graphene_sqlalchemy/tests/test_batching.py

```diff
@@ -608,8 +663,8 @@ def resolve_reporters(self, info):
 
 
 @pytest.mark.asyncio
-def test_batch_sorting_with_custom_ormfield(session_factory):
-    session = session_factory()
+def test_batch_sorting_with_custom_ormfield(sync_session_factory):
+    session = sync_session_factory()
```

Comment:
Sync test marked as asyncio, but I think that wasn't due to your change

Reply:
Nope it wasn't 😉 Removed it anyhow 👍